CMakeLists.txt

```diff
@@ -38,7 +42,7 @@ file(GLOB_RECURSE Generic
 add_subdirectory(pdclib)
 add_subdirectory(doom)
 add_subdirectory(ArtOSTypes)
-add_subdirectory(helloworld)
+#add_subdirectory(helloworld)
```

Comment:
Removed because it doesn't compile yet (I need to write the linker files (crt0.o,crtn.o and crti.o)